### PR TITLE
Disappearing Platforms

### PR DESCRIPTION
**Pull request by enbaik**
Adding platform prefab that disappears after player jumps off to add to level difficulty.

**Estimated:30 Min:15 Max:60 Actual: 45

### CODE REVIEW

File: 385/Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -132,6 +132,8 @@ public bool IsOnGround
 	/// </summary>
 	private Vector3 playerPos;
 
+    private
```
Comment (Chris-Johnston) [inline 385/Assets/Scripts/PlayerController.cs]: start should not be `private`, and this is not clean code.

File: 385/Assets/Scripts/Tags.cs [hunk 2]
```diff
@@ -14,4 +14,5 @@ public static class Tags {
 	public const string TAG_GROUND = "Ground";
 	public const string TAG_GRAPPLE_POINT = "GrapplePoint";
 	public const string TAG_GRAPPLE_HOOK = "GrappleHook";
+    public const string TAG_GROUND_DIS = "GroundDis";
```
Comment (Chris-Johnston) [inline 385/Assets/Scripts/Tags.cs]: please use consistent formatting

Comment (enbaik) [reply 385/Assets/Scripts/Tags.cs]: TAG_GROUND_DISAPPEARING = "GroundDisappearing" ?

Comment (Chris-Johnston) [reply 385/Assets/Scripts/Tags.cs]: No, indentation

Comment (enbaik) [reply 385/Assets/Scripts/Tags.cs]: that's odd, the indentation is the right when i opened it, i'm going to re-enter it just beneath TAG_GROUND and see if it works

**Comment by Chris-Johnston** [comment]
Works as intended, but I still like the idea of platforms that self destruct after a few seconds. Should consider this for another PR down the time if we have time, and definitely after #89 is merged.

LGTM :shipit:

**Comment by SaamIAm** [comment]
Looks good, interesting thing I noticed If you jump into a disappearing platform from underneath it disappears. Works how I expected it to.

**Pull request merged**